src/App.vue

```diff
@@ -32,7 +32,7 @@ Vue.component('fa', FontAwesomeIcon)
 })
 export default class App extends Vue {
   defaultChainID = 'default'
-  allowedUrls = ['http://127.0.0.1:46658']
+  allowedUrls = ['https://plasma.dappchains.com']
```

Comment:
I think the plasma url should be the first element in `allowedUrls` so it's the default, but we probably still want to keep `127.0.0.1` as one of the options.

Reply:
That's make sense !